x/gov/client/cli/prompt.go

```diff
@@ -155,6 +155,10 @@ func (p *proposalType) Prompt(cdc codec.Codec) (*proposal, types.ProposalMetadat
 		return nil, metadata, fmt.Errorf("failed to set proposal deposit: %w", err)
 	}
 
+	// set title and summary from metadata in the proposal
```

Comment:
Nit actually we can move that up to L147, just after we set the proposal.Metadata. IMHO it makes more sense to group that.

Maybe even assign all three of them in the struct directly and remove L138.

```go
proposal := &proposal{
    Metadata: "IPFS", // comment
    Title: metadata.Title,
    Summary: metadata.Summary, 
}
```